.github/workflows/E2Etests.yml

```diff
@@ -9,46 +9,49 @@ jobs:
       matrix:
         node: ["14"]
         browser: ["firefox", "chrome"]
+
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+
     steps:
       - uses: actions/checkout@v2
-      - name: Setup and Run backend
-        env: 
-          DEVELOPMENT: true
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          driver-opts: network=host
+
+      - name: Clone backend
+        uses: actions/checkout@v2
+        with:
+          repository: 'CSCfi/beacon-network'
+          ref: 'feature/multistage-build'
```

Comment:
when back-end https://github.com/CSCfi/beacon-network/pull/84 is merged to `dev` this will be changed